src/propodesolvers.jl

```diff
@@ -31,7 +31,8 @@ for (qu_ode_type,ode_solver) in type_to_method_ode
             dims = size(current_qustate)
             # Convert the current_qustate to complex as it might result in a Inexact Error. After complex is in QuBase.jl (PR #38)
             # we could just do a complex(vec(current_qustate)) avoiding the coeffs(coeffs(vec(current_qustate))).
-            next_state = $ode_solver((t,y)-> -im*coeffs(op)*y, complex(coeffs(vec(current_qustate))), [current_t, t], points=:specified,
+            f = @anon (t,y)-> -im*coeffs(op)*y
```

Comment:
You can try to do `scale!(coeffs(op)*y, -im)`, which should save one copy and some multiplications.